web-api/src/persistence/postgres/polling/schema.ts

```diff
@@ -0,0 +1,44 @@
+import { Selectable, Insertable, Updateable } from 'kysely';
+
+const DEFAULT = {};
+
+export const requestTableDefinition = {
+  requestId: DEFAULT as string,
+  totalChunks: DEFAULT as number | null,
+  userId: DEFAULT as string,
+  ttl: DEFAULT as number,
+};
+
+export type RequestTable = typeof requestTableDefinition;
+export const DW_REQUEST_COLUMNS = Object.keys(requestTableDefinition) as Array<
+  keyof RequestTable
+>;
+
+export type RequestKysely = Selectable<RequestTable>;
+export type NewRequestKysely = Insertable<RequestTable>;
+export type UpdateRequestKysely = Updateable<RequestTable>;
+
+export const responseChunkTableDefinition = {
+  chunk: DEFAULT as string,
+  index: DEFAULT as number,
+  requestId: DEFAULT as string,
+  totalNumberOfChunks: DEFAULT as number,
+  ttl: DEFAULT as number,
+  userId: DEFAULT as string,
+};
+
+export type ResponseChunkTable = typeof responseChunkTableDefinition;
```

Comment:
Were we chunking because of the Dynamo record size limit of 400KB? If so, I think Postgres allows a field to be 1GB, so I wonder if we need to chunk at all.

Reply:
Based on the chunking I see in lambdaWrapper, my suspicion is that _is_ why we were chunking. Might be worth consulting with others to see if it is even needed now.